apps/chat/channels.py

```diff
@@ -587,6 +589,34 @@ def send_voice_to_user(self, synthetic_voice: SynthesizedAudio):
         )
 
 
+class SureAdhereChannel(ChannelBase):
+    def initialize(self):
+        self.messaging_service = self.experiment_channel.messaging_provider.get_messaging_service()
+
+    def send_text_to_user(self, text: str):
+        to_patient = self.chat_id
+        self.messaging_service.send_text_message(text, to=to_patient, platform=ChannelPlatform.IN_APP)
+
+    def get_chat_id_from_message(self, message):
+        return message.chat_id
+
+    @property
+    def voice_replies_supported(self) -> bool:
+        return bool(settings.AWS_ACCESS_KEY_ID) and self.messaging_service.voice_replies_supported
```

Comment:
It seems like voice isn't supported so you could leave this property out since the default is `False`